minigui/log.ts

```diff
@@ -0,0 +1,71 @@
+import {getElement, querySelector} from './util';
+
+function escapeAnsiColors(raw: string) {
```

Comment:
probably better to axe this and just send the non-colored version of the board over the wire, no?